experiments/20160805-mutagenesis/20160902-mutagenesis-pb2.md

```diff
@@ -36,3 +36,16 @@
 | 72                   | 2 min          | times    |
 | 72                   | 10 min         | 1        |
 | 4                    | hold           |          |
+
+### Gel Electrophoresis
+
+Gel Order:
+
+| Ladder | Hi | Lo | W | T | Ladder |
+|--------|----|----|----|-----|--------|
+
+![gel](./20160905-pb2_full.jpg)
+
+## Analysis
+
+Mutant PB2 reactions yielded no difference in products from template at all.
```

Comment:
Great try! We at least know that the 30/26 combination won't work in the future, which thus makes it necessary to do the split PCRs.. :smile: